app/scripts/controllers/transactions/index.js

```diff
@@ -422,6 +422,7 @@ export default class TransactionController extends EventEmitter {
       gasLimit: defaultGasLimit,
       simulationFails,
     } = await this._getDefaultGasLimit(txMeta, getCodeResponse);
+    const advancedGasFeeDefaultEIP1559V2 = this.getAdvancedGasFee();
```

Comment:
Will this always stay `V2`? Just thinking about what we would have to do when this feature goes live — it would be nice if there were as few places as possible where we'd have to remove "V2". Maybe just `advancedGasFeeDefaults`?